docs/docs/guides/rust.md

```diff
@@ -1,3 +1,3 @@
-# Writing DBSP programs in Rust
+# Writing Feldera programs in Rust
```

Comment:
This should definitely be DBSP

Reply:
Yes, this one is DBSP. And the title should probably be "Using the DBSP Rust API".